tests/css_atrule/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1,98 +1,2394 @@
+  (sea-slug, blue, pointer), (egret, white, move) {
+}
+@each $animal, $color, $cursor in (puma, black, default),
+   (sea-slug, blue, pointer),  (egret, white, move) {
+}
+
+`;
+
+exports[`extend.css 1`] = `
+.master {
+    color:  black;
+    font-size: 12px;
+}
+.emphasis {
+    @extend .master;
+    font-weight: bold;
+}
+.emphasis {
+@extend .master;
+font-weight: bold;
+}
+.emphasis {
+    @extend  .master ;
+    font-weight: bold;
+}
+.emphasis {
+    @extend   .master  ;
+    font-weight: bold;
+}
+.emphasis
+{
+@extend
+.master
+;
+font-weight
+:
+bold
+;
+}
+.emphasis
+
+{
+
+@extend
+
+.master
+
+;
+
+font-weight
+
+:
+
+bold
+
+;
+
+}
+%message-shared {
+    border: 1px solid #ccc;
+    padding: 10px;
+    color: #333;
+}
+.message {
+    @extend %message-shared;
+}
+.message {
+@extend %message-shared;
+}
+.message {
+    @extend  %message-shared ;
+}
+.message {
+    @extend   %message-shared  ;
+}
+.message {
+@extend
+%message-shared;
+}
+.message {
+
+@extend
+
+%message-shared;
+
+}
+a.important {
+    @extend .notice !optional;
+}
+a.important {
+@extend .notice !optional;
+}
+a.important {
+    @extend  .notice !optional ;
+}
+a.important {
+    @extend  .notice  !optional  ;
+}
+a.important
+{
+@extend
+.notice
+!optional
+;
+}
+a.important
+
+{
+
+@extend
+
+.notice
+
+!optional
+
+;
+
+}
+.message-error {
+    @extend .message, .important;
+}
+.message-error {
+    @extend .message , .important ;
+}
+.message-error {
+    @extend   .message  ,  .important  ;
+}
+.message-error {
+    @extend .message,
+        .important;
+}
+.message-error {
+    @extend
+        .message,
+        .important;
+}
+.message-error
+{
+@extend
+.message
+,
+.important
+;
+}
+.message-error
+
+{
+
+@extend
+
+.message
+
+,
+
+.important;
+
+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.master {
+  color: black;
+  font-size: 12px;
+}
+.emphasis {
+  @extend .master;
+  font-weight: bold;
+}
+.emphasis {
+  @extend .master;
+  font-weight: bold;
+}
+.emphasis {
+  @extend .master;
+  font-weight: bold;
+}
+.emphasis {
+  @extend .master;
+  font-weight: bold;
+}
+.emphasis {
+  @extend .master;
+  font-weight: bold;
+}
+.emphasis {
+  @extend .master;
+
+  font-weight: bold;
+}
+%message-shared {
+  border: 1px solid #ccc;
+  padding: 10px;
+  color: #333;
+}
+.message {
+  @extend %message-shared;
+}
+.message {
+  @extend %message-shared;
+}
+.message {
+  @extend %message-shared;
+}
+.message {
+  @extend %message-shared;
+}
+.message {
+  @extend %message-shared;
+}
+.message {
+  @extend %message-shared;
+}
+a.important {
+  @extend .notice !optional;
+}
+a.important {
+  @extend .notice !optional;
+}
+a.important {
+  @extend .notice !optional;
+}
+a.important {
+  @extend .notice !optional;
+}
+a.important {
+  @extend .notice !optional;
+}
+a.important {
+  @extend .notice !optional;
+}
+.message-error {
+  @extend .message,
+    .important;
```

Comment:
```scss
@extend .message, .important;
```

Reply:
@j-f1 yep, also will add test when selector is very long

Reply:
@j-f1 done